src/app/components/drawer/Projects/index.js

```diff
@@ -15,7 +15,7 @@ const renderQuery = ({ error, props }) => {
         currentUser={props.me}
         team={props.team}
         savedSearches={props.team.saved_searches.edges.map(ss => ss.node)}
-        feeds={feeds.map(f => ({ ...f, title: (f.name || f.feed?.name) }))}
+        feeds={feeds.map(f => ({ ...f, title: (f.name || f.feed?.name), dbid: (f.feed_id || f.dbid) }))}
```

Comment:
Maybe just add a comment here pointing out that sometimes these are `Feed` objects and sometimes these are `FeedInvitation` objects.